symfony/apps/orangehrm/templates/freshorange.php

```diff
@@ -1,24 +1,34 @@
-<?php 
+<?php
 
 // Allow header partial to be overridden in individual actions
 // Can be overridden by: slot('header', get_partial('module/partial'));
 include_slot('header', get_partial('global/header'));
+$subscribed = $sf_user->isSubscribed();
 ?>
 
     </head>
     <body>
-      
+
         <div id="wrapper">
-            
+
             <div id="branding">
                 <a href="http://www.orangehrm.com/" target="_blank"><img src="<?php echo theme_path('images/logo.png')?>" width="283" height="56" alt="OrangeHRM"/></a>
-                <!--<a href="http://www.orangehrm.com/user-survey-registration.php" class="subscribe" target="_blank"><?php echo __('Join OrangeHRM Community'); ?></a>-->
                 <a href="#" id="welcome" class="panelTrigger"><?php echo __("Welcome %username%", array("%username%" => $sf_user->getAttribute('auth.firstName'))); ?></a>
-                <div id="MP_btn">
-                    <input id="MP_link" type="button" class="button" value="<?php echo __('Marketplace'); ?>"/>
-                </div>
+                <?php if(!$subscribed) { ?>
+                    <div id="MP_btn">
+                        <input id="Subscriber_link" type="button" class="button" value="<?php echo __('Subscribe'); ?>"/>
+                    </div>
+                    <div style="float: right; padding-top: 58px;">
```

Comment:
may be we can add a class or an id for the element and move this inline styles to main css or some existing css file.

Reply:
done